docs/examples/generatedCanonicalFields.json

```diff
@@ -0,0 +1,822 @@
+[
+  {
+    "label": "Org Name",
+    "shortCode": "Org Name",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Mission Statement",
+    "shortCode": "Org Mission Statement",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Website",
+    "shortCode": "Org Website",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Entity Type",
+    "shortCode": "Org Entity Type",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Registration Number: EIN / TIN",
+    "shortCode": "Org Registration Number: EIN / TIN",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Address",
+    "shortCode": "Org Address",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Phone",
+    "shortCode": "Org Phone",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Email",
+    "shortCode": "Org Email",
+    "dataType": "string"
+  },
+  {
+    "label": "Org DBA Name",
+    "shortCode": "Org DBA Name",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Banking Information",
+    "shortCode": "Org Banking Information",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Legal Docs",
+    "shortCode": "Org Legal Docs",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Board Members",
+    "shortCode": "Org Board Members",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Administrative Lead",
+    "shortCode": "Org Administrative Lead",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Annual Operating Budget",
+    "shortCode": "Org Annual Operating Budget",
+    "dataType": "number"
```

Comment:
These types are JSON types. If the type does not appear to be an integer, number, boolean, then it becomes a string.

Reply:
Specifically these, right? https://json-schema.org/understanding-json-schema/reference/type.html

Reply:
I had been going for pure JSON types rather than JSON schema types but JSON schema types would be better. I think the key difference is whether integer is recognized as integer or number, right?

Reply:
I believe so - yeah why don't we plan for that; heck in some future iteration we could even have our "type" field here contain a full blown json schema definition (which could even help solve our validation question / problem!)

Reply:
We now keep `integer` as `integer`.